docs/examples/viz_slicer.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
```

Comment:
There is a high probability that we will never come back to this Tutorial. Or in a very long time....

So, this tutorial needs much more explanation. It needs a header + title, like in DIPY.